### PR TITLE
minor: allow feedback email to be configured via JSON

### PR DESCRIPTION
**Pull request by jenbutongit**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Please also include any acceptance criteria if you have any.

Currently the feedback banner returns empty if the feedback object is empty. 

- allow feedback.email to be configured (via JSON only) 
  - accepts any valid email
  - then appends `mailto:` for the banner

No integration with the designer for now, since that page already has a bug which prevents any saves occurring (schema validation fails) #293.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce
the testing if necessary.

Before PR's can be merged they will need to be tested by QA and approved where
applicable. To flag the change to QA assign **@XGovFormBuilder/qa** as one of the reviewers.

- [x] Unit 
- [x] manual

# Checklist:

- [x] My changes do not introduce any new linting errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and versioning
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have rebased onto main and there are no code conflicts
- [n/a] I have checked deployments are working in all environments
- [x] I have updated the architecture diagrams as per Contribute.md

### CODE REVIEW

File: model/src/schema/schema.ts [hunk 1]
```diff
@@ -169,6 +169,14 @@ const feedbackSchema = joi.object().keys({
     is: joi.boolean().valid(false),
     then: joi.string().optional(),
   }),
+  email: joi
```
Comment (superafroman) [inline model/src/schema/schema.ts:172]: I'll never win .. an email is the thing that's sent to an email address :)

**Review by superafroman — DISMISSED** [review]
What benefit does this bring over just putting a mail to link in the original feedbackLink field?

**Comment by jenbutongit** [comment]
I guess this is partly a bug fix as well..

FEEDBACK_LINK env variable isn't used for form level anymore. The feedback object in the JSON allows you to this though. If feedback.url isn't a relative URL it will throw an error (I tried that first), so I added feedback.email so we can do the right validation on it

File: model/src/schema/schema.ts (hunk 1 above)
Comment (jenbutongit) [reply model/src/schema/schema.ts:172]: That's my bad :p.. changing. Who says I never let you win 🏆

**Review by superafroman — APPROVED** [review]

**Pull request merged**